packages/flutter/lib/src/material/text_field.dart

```diff
@@ -959,8 +959,7 @@ class _TextFieldState extends State<TextField> implements TextSelectionGestureDe
   @override
   Widget build(BuildContext context) {
     assert(debugCheckHasMaterial(context));
-    // TODO(jonahwilliams): uncomment out this check once we have migrated tests.
-    // assert(debugCheckHasMaterialLocalizations(context));
+    assert(debugCheckHasMaterialLocalizations(context));
```

Comment:
@jonahwilliams is there an issue to reference here? I don't think this is ready to be uncommented.